packages/box/box.1.2.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Render boxes in the terminal"
+description: """
+Render boxes in the terminal, can configure the padding, margin and the border of the box.
+"""
+maintainer: ["davesnx"]
+authors: ["davesnx <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/davesnx/ocaml-box"
+doc: "https://davesnx.github.io/ocaml-box/box/index.html"
+bug-reports: "https://github.com/davesnx/ocaml-box/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7" & >= "2.7"}
```

Comment:
To remove this you need to remove the dependency on dune in the dune-project file
```suggestion
  "dune" {>= "2.7"}
```

Reply:
Alternatively you should not need the opam file autogeneration from dune so you can turn it off and modify the opam file directly instead.